plugins/release-dependencies-diff/src/main/kotlin/ReleaseDependenciesDiffFilesTask.kt

```diff
@@ -0,0 +1,102 @@
+import org.gradle.api.DefaultTask
+import org.gradle.api.file.RegularFileProperty
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.OutputFile
+import org.gradle.api.tasks.TaskAction
+import org.gradle.api.tasks.options.Option
+import org.gradle.kotlin.dsl.support.appendReproducibleNewLine
+import java.io.ByteArrayOutputStream
+import java.io.File
+
+abstract class ReleaseDependenciesDiffFilesTask : DefaultTask() {
+
+    @get:Input
+    @set:Option(
```

Comment:
Cool approach with `Option`.
Not sure how if it `UP-TO-DATE` if contents of provided paths are changed, but probably we do not need it right now (which is wrong).